bindings/php/opendal-php.stubs.php

```diff
@@ -20,6 +20,111 @@
 
 // Stubs for opendal-php
 
-namespace {
-    function debug(): string {}
+namespace OpenDAL {
+    class Operator {
+        public function __construct(string $scheme_str, array $config) {}
+
+        /**
+         * Write bytes into given path.
+         */
+        public function write(string $path, string $content): mixed {}
+
+        /**
+         * Read the whole path into bytes.
+         */
+        public function read(string $path): string {}
```

Comment:
The returning content could also be bytes.

Reply:
done